src/Services/DeviceService.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * DeviceService.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Database\QueryUtils;
+use OpenEMR\Common\Logging\SystemLogger;
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Services\Search\FhirSearchWhereClauseBuilder;
+use OpenEMR\Validators\ProcessingResult;
+
+class DeviceService extends BaseService
+{
+    private const DEVICE_TABLE = "lists";
+    private $uuidRegistry;
+
+    public function __construct()
+    {
+        parent::__construct('lists');
+        $this->uuidRegistry = new UuidRegistry(['table_name' => self::DEVICE_TABLE]);
+        $this->uuidRegistry->createMissingUuids();
```

Comment:
prob should also do for the patient_data table

Reply:
This should be fixed now.

Reply:
gotta run createMissingUuid on each one :)
Would prob just do following in case need the main $this->uuidRegistry in the class.
```
$this->uuidRegistry = new UuidRegistry(['table_name' => self::DEVICE_TABLE]);
$this->uuidRegistry->createMissingUuids();
(new UuidRegistry(['table_name' => self::PATIENT_TABLE]))->createMissingUuids();
```